app/lib/activitypub/activity/create.rb

```diff
@@ -61,7 +61,7 @@ def process_status_params
         account: @account,
         text: text_from_content || '',
         language: detected_language,
-        spoiler_text: text_from_summary || '',
+        spoiler_text: converted_object_type? ? '' : (text_from_summary || ''),
```

Comment:
this should be object[:sensitive] or something equivalent, so converted object types can still opt in to CWs